e2e/cypress/integration/channel_sidebar/more_unreads_position_with_scroll_spec.js

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
This test belongs to "Channel Settings" functional area. Therefore, instead of "channel_sidebar" folder, please create a new "channel_settings" folder and move the spec file there.